packages/online-editor/it-tests/support/commands.ts

```diff
@@ -47,12 +47,15 @@ Cypress.Commands.add("loadEditor", () => {
   });
 });
 
-Cypress.Commands.add("ouiaId", { prevSubject: "optional" }, (subject, id: string, options = {}) => {
-  const idSelector = `[data-ouia-component-id='${id}']`;
+Cypress.Commands.add("ouia", { prevSubject: "optional" }, (subject, locator, options = {}) => {
```

Comment:
Just checking cypress commands here.

As long as IDE-based hinting works for the selector attribute I would consider this a superior solution to what we have elsewhere:
In kogito-apps we have 2 commands: ouiaId, ouiaType, each one for different mode of operation. So this raises a question if generally `ouia({ouiaId:'xy'})` is better than `ouiaId('xy')`, or more complex `ouia({ouiaId:'xy', ouiaType:'ab'})` compared to `ouiaId('xy', 'ab')`. Which I think it is (cause in the other solution the order of arguments is not clear).
For reference: https://github.com/kiegroup/kogito-apps/blob/main/ui-packages/packages/cypress-ouia/src/index.ts

But generally I like the approach here - when we're merging those cypress-ouia tools globally, your approach would be the one to adopt.

Reply:
Thanks @jomarko for checking the IDE hinting support, it seems fine, so no problem with this command.